src/stats.hpp

```diff
@@ -28,45 +28,45 @@ class Config;
 class Context;
 
 // Statistics fields in storage order.
-enum stats {
-  STATS_NONE = 0,
-  STATS_STDOUT = 1,
-  STATS_STATUS = 2,
-  STATS_ERROR = 3,
-  STATS_CACHEMISS = 4,
-  STATS_PREPROCESSOR = 5,
-  STATS_COMPILER = 6,
-  STATS_MISSING = 7,
-  STATS_CACHEHIT_CPP = 8,
-  STATS_ARGS = 9,
-  STATS_LINK = 10,
-  STATS_NUMFILES = 11,
-  STATS_TOTALSIZE = 12,
-  STATS_OBSOLETE_MAXFILES = 13,
-  STATS_OBSOLETE_MAXSIZE = 14,
-  STATS_SOURCELANG = 15,
-  STATS_BADOUTPUTFILE = 16,
-  STATS_NOINPUT = 17,
-  STATS_MULTIPLE = 18,
-  STATS_CONFTEST = 19,
-  STATS_UNSUPPORTED_OPTION = 20,
-  STATS_OUTSTDOUT = 21,
-  STATS_CACHEHIT_DIR = 22,
-  STATS_NOOUTPUT = 23,
-  STATS_EMPTYOUTPUT = 24,
-  STATS_BADEXTRAFILE = 25,
-  STATS_COMPCHECK = 26,
-  STATS_CANTUSEPCH = 27,
-  STATS_PREPROCESSING = 28,
-  STATS_NUMCLEANUPS = 29,
-  STATS_UNSUPPORTED_DIRECTIVE = 30,
-  STATS_ZEROTIMESTAMP = 31,
-  STATS_CANTUSEMODULES = 32,
+enum class Stats {
```

Comment:
> > had to apply formatting again since INTERNAL_ERROR has some additional chars... 🤣
> 
> Tip: Set up your editor to run clang-format automatically on save. To be able to automatically format code on save is the primary reason I want to use clang-format – it's just so convenient.

Funny enough big old clumsy Visual Studio does that. The new and shiny Visual Studio Code does not without some plugin.